app/config/doctrine.yml

```diff
@@ -13,6 +13,7 @@ doctrine:
         user: "%database_user%"
         password: "%database_password%"
         charset: utf8mb4
+        server_version: 8
```

Comment:
Why this change ?

Reply:
A left-over. Thanks for the report! It is fixed now.